templates/build-pod/template.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
Can we name this template `maven-build-pod` (or `mvn-build-pod` ) instead of the generic `build-pod`?